cli/tests/pcluster/validators/test_all_validators/test_slurm_validators_are_called_with_correct_argument/slurm.yaml

```diff
@@ -52,11 +53,31 @@ Scheduling:
         - Name: compute_resource3
           InstanceType: t2.large
           DisableSimultaneousMultithreading: true
+          CapacityReservationTarget:
+            CapacityReservationId: "cr-34567"
       Tags:
         - Key: queue_tag1
           Value: String
         - Key: queue_tag2
           Value: String
+    - Name: queue3
+      CapacityType: CAPACITY_BLOCK
+      Networking:
+        SubnetIds:
+          - subnet-23456789
+      ComputeResources:
+        - Name: compute_resource1
+          InstanceType: t2.xlarge
+          MinCount: 5
+          MaxCount: 5
+          CapacityReservationTarget:
+            CapacityReservationId: "cr-12345"
+        - Name: compute_resource2
+          InstanceType: t2.xlarge
+          MinCount: 3
+          MaxCount: 3
+          CapacityReservationTarget:
```

Comment:
what about adding a "CapacityReservationTarget" at queue level? So to check validation are correctly called for that case too

Reply:
done, moved it.